README.md

```diff
@@ -8,5 +8,5 @@ Shell scripts, ansible scripts and config files for foss united servers and self
 - [Nextcloud](https://nextcloud.com) - Experimenting with nextcloud for switching from the GSuite (like GDoc, GSpreadsheets)
 - [Vaultwarden](./servers/apps/vaultwarden/README.md) - Password Management on Organization and User level
 - [Mattermost](./servers/apps/mattermost/mattermost.md) - For Internal team discussions
-- [Librechat](https://www.librechat.ai/) - Custom ChatGPT Web UI called [Echo](https://echo.fossunited.org)
+- [Librechat](https://www.librechat.ai/) - AI chat platform/search engine called [Echo](https://echo.fossunited.org)
```

Comment:
>/search engine 

I'm unsure in this part. Checked https://librechat.ai and it does not say that it is also a search engine ?

Reply:
We are using it as a prompt based search engine too isnt it? I just mentioned the usecase we do for it.